website/public/js/controllers/inviteToGroupCtrl.js

```diff
@@ -26,6 +26,11 @@ habitrpg.controller('InviteToGroupCtrl', ['$scope', 'User', 'Groups', 'injectedG
       _inviteByMethod(inviteMethod);
     };
 
+    $scope.getMemberAndInviteCount = function(group) {
+
+      return group.memberCount + group.invites.length;
```

Comment:
@colwem it's possible group.invites is undefined here? Try adding a check for it.

`return group.memberCount + (group.invites ? group.invites.length : 0)`